backend/app/settings.py

```diff
@@ -0,0 +1,32 @@
+import secrets
+
+from pydantic import BaseSettings
+
+
+class Settings(BaseSettings):
```

Comment:
Please add a comment saying values for `OIDC_PROVIDER` and `PRODUCT_GATEWAY_URL` (why are the name patterns different?) can be determined from the `.well-known` configuration for the dataspace + link to the docs